src/airbrake.cr

```diff
@@ -49,22 +49,29 @@ module Airbrake
   class Config
     property :project_id
     property :project_key
+    property :host
+    property :port
+    property :secure
+
+    def uri
+      self.host   ||= "airbrake.io"
+      self.port   ||= 443
+      scheme = (self.secure.nil? || self.secure) ? "https" : "http"
+
+      URI.new(scheme, host, port, "/api/v3/projects/#{project_id}/notices", "key=#{project_key}").to_s
```

Comment:
I'm not a fan of reassigning the host property. If you want to reduce the number of properties I would suggest a different name.
If we introduce the "endpoint" property and remove "host", "port" and "secure" we can just combine the endpoint and the path. However URI.join does not exists (yet) so we have to do that differently.